src/controllers/expenses.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-fallthrough */
+'use strict';
+
+const {
+  getAll,
+  getById,
+  create,
+  remove,
+  update,
+  getAllForUser,
+  getAllForCategory,
+  getAllBetweenDates,
+} = require('../services/expenses');
+
+const { getById: getUserById } = require('../services/users');
+
+const getAllExpenses = (req, res) => {
+  let expenses = getAll();
+  const { userId } = req.query;
+  const { categories } = req.query;
+  const { from, to } = req.query;
+
+  switch (true) {
+    case !!userId:
+      if (userId) {
+        expenses = getAllForUser(userId);
+      }
+
+    case !!categories:
+      if (categories) {
```

Comment:
Remove unnecessary 'if (categories)' check, as it is already checked in the switch case.

Reply:
Done